x-pack/test/api_integration/deployment_agnostic/services/alerting_api.ts

```diff
@@ -192,7 +1089,7 @@ export function AlertingApiProvider({ getService }: DeploymentAgnosticFtrProvide
       return body;
     },
 
-    async findRule(ruleId: string, roleAuthc: RoleCredentials) {
+    async findRuleInRules(roleAuthc: RoleCredentials, ruleId: string) {
       const response = await supertestWithoutAuth
         .get('/api/alerting/rules/_find')
         .set(roleAuthc.apiKeyHeader)
```

Comment:
I'm a bit confused by the naming, but I saw there is another `findRule` function and I think it was confusing initially.

Since you anyways work on the service, what do you think about the following:

findRule -> findRuleById(ruleId: string, roleAuthc: RoleCredentials)
when `/api/alerting/rule/${ruleId}` is used in API call

findRule -> findInRules(ruleId: string, roleAuthc: RoleCredentials) (or `findInAllRules`)
when `/api/alerting/rules/_find` is used in API call and later we filter response

Reply:
Yeah there were two originally, due to merging the helpers
It's like the helpers were quickly made, instead of doing what we are doing now: discussing where they fit best, not just fit...good enough to compile and run